flexmeasures/ui/templates/accounts/account.html

```diff
@@ -268,7 +268,7 @@ <h3>Assets
       {% if can_view_account_auditlog %}
       <button
         class="btn p-3 btn-info border-0 mb-3 mt-3" type="button"
-        onclick="window.location.href='/users/auditlog/{{ account.id }}'"
+        onclick="window.location.href='/accounts/auditlog/{{ account.id }}'"
```

Comment:
Wouldn't this have been more RESTful?
```
onclick="window.location.href='/accounts/{{ account.id }}/auditlog'"
```

Reply:
Yes, but that is not the quest of this PR.